source/en/sdr.rst

```diff
@@ -465,39 +465,39 @@ Interpreting Results
 
     * **Parameter log**: Each time the model is run, a text (.txt) file will be created in the Workspace. This file will list the parameter values and output messages for that run and will be named according to the service, the date and time, and the suffix. When contacting NatCap about errors in a model run, please include the parameter log.
 
-    * **rkls.tif** (type: raster; units: tons/pixel/year): Total potential soil loss per pixel in the original land cover from the RKLS equation. Equivalent to the soil loss for bare soil. (Eq. :eq:`usle`, without applying the :math:`C` or :math:`P` factors)
+    * **rkls.tif** (type: raster; units: metric tons/hectare/year): Total potential soil loss in the original land cover from the RKLS equation. Equivalent to the soil loss for bare soil. (Eq. :eq:`usle`, without applying the :math:`C` or :math:`P` factors)
 
-    * **sed_export.tif** (type: raster; units: tons/pixel/year): The total amount of sediment exported from each pixel that reaches the stream. (Eq. :eq:`e_i`)
+    * **sed_export.tif** (type: raster; units: metric tons/hectare/year): The total amount of sediment exported from each pixel that reaches the stream. (Eq. :eq:`e_i`)
 
-    * **sed_deposition.tif** (type: raster; units: tons/pixel/year): The total amount of sediment deposited on the pixel from upslope sources as a result of trapping. (Eq. :eq:`ti`)
+    * **sed_deposition.tif** (type: raster; units: metric tons/hectare/year): The total amount of sediment deposited on the pixel from upslope sources as a result of trapping. (Eq. :eq:`ti`)
 
     * **stream.tif** (type: raster): Stream network, created using flow direction and flow accumulation derived from the DEM and Threshold Flow Accumulation. Values of 1 represent streams, values of 0 are non-stream pixels. Compare this layer with a real-world stream map, and adjust the Threshold Flow Accumulation so that this map matches real-world streams as closely as possible. See the User Guide section :ref:`working-with-the-DEM` for more information.
 
     * **stream_and_drainage.tif** (type: raster): If a drainage layer is provided, this raster is the union of that layer with the calculated stream layer(Eq. :eq:`stream_and_drainage`). Values of 1 represent streams, values of 0 are non-stream pixels.
 
-    * **usle.tif** (type: raster; units: tons/pixel/year): Total potential soil loss per pixel in the original land cover calculated from the USLE equation. (Eq. :eq:`usle`)
+    * **usle.tif** (type: raster; units: metric tons/hectare/year): Total potential soil loss in the original land cover calculated from the USLE equation. (Eq. :eq:`usle`)
 
-    * **avoided_erosion.tif** (type: raster; units: tons/pixel/year): The contribution of vegetation to keeping soil from eroding from each pixel. (Eq. :eq:`aer_i`)
+    * **avoided_erosion.tif** (type: raster; units: metric tons/hectare/year): The contribution of vegetation to keeping soil from eroding from each pixel. (Eq. :eq:`aer_i`)

```

Comment:
```suggestion
    * **avoided_erosion.tif** (type: raster; units: t/(ha*year)): The contribution of vegetation to keeping soil from eroding from each pixel. (Eq. :eq:`aer_i`)

```

Reply:
My usual inclination is to spell things out as much as possible, and consistently. So personally I'd be more in favor of "metric tons/(hectare*year)" (or "metric tons/hectare/year", which reads a little more clearly to me, but let's pick one way of writing it and do it consistently throughout the UG).  If I was a beginner with the models, I'm not sure I'd know what "t" stands for.